exceptioncontext.py,sha256=UbBOxAaAIbDqLZEHcPwt3o70UIOLDKbKZ0_7eBbujsg,2462
exceptioncontext-1.0.5.dist-info/LICENSE,sha256=gTW0XSdjEa1mV_oHlGPe72A66yFrkDs4-BFF06W6vbw,671
exceptioncontext-1.0.5.dist-info/METADATA,sha256=QWLGVGl62m-IQpGFk-BAuL6Pfxj5xNK-NdQYmYivdO0,2750
exceptioncontext-1.0.5.dist-info/WHEEL,sha256=h_aVn5OB2IERUjMbi2pucmR_zzWJtk303YXvhh60NJ8,110
exceptioncontext-1.0.5.dist-info/top_level.txt,sha256=fCGjpWqKY0BrfagW2z4p2s5YMthziA0lqb9GOuNqmF0,17
exceptioncontext-1.0.5.dist-info/RECORD,,
